canonical-web-and-design/charmhub.io
# Removed temporary changes, append cards to top related

## Done
- Removed changes from previous temporary fixes
- Fill up topics to ensure there are always 3 displayed - with the most relevant first

## How to QA
- Visit the demo in your browser, link commented below
  - Or you can run the project locally using the [dotrun](https://snapcraft.io/dotrun) snap with `$ dotrun` and view it in your web browser at: http://localhost:8045/
  - If your demo doesn't work, read more about the [demoservice](https://discourse.ubuntu.com/t/demoservice/16862)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Visit https://charmhub-io-$.demos.haus, filter by the categories
- See that there is always 3 topics

## Issue / Card
Fixes #1289

## Screenshots
![image](https://user-images.githubusercontent.com/479384/159015091-3afcff3a-7963-40d1-aa3f-707ea2e1f923.png)
